themes/src/main/resources/theme/base/account/messages/messages_en.properties

```diff
@@ -122,7 +122,7 @@ invalidEmailMessage=Invalid email address.
 missingLastNameMessage=Please specify last name.
 missingEmailMessage=Please specify email.
 missingPasswordMessage=Please specify password.
-notMatchPasswordMessage=Passwords don''t match.
```

Comment:
Java properties require ' to be escaped, hence why it's '' and not just '

Reply:
Sorry, I have misunderstood. I will remove them.